tests/platform_tests/api/test_watchdog.py

```diff
@@ -47,37 +50,49 @@ def conf(self, request, duthost):
 
         if platform in test_config and 'default' in test_config[platform]:
             config.update(test_config[platform]['default'])
-
         if platform in test_config and hwsku in test_config[platform]:
             config.update(test_config[platform][hwsku])
 
-        assert 'valid_timeout' in config
+        self.expect('valid_timeout' in config, "valid_timeout is not defined in config")
```

Comment:
You can just use `pytest_assert()` here. No point in continuing on if this fails.